net/haproxy/src/opnsense/service/templates/OPNsense/HAProxy/haproxy.conf

```diff
@@ -534,11 +541,19 @@
     # ACL ERROR COUNT: {{acl_errors}}
 {%       endif %}
 {%     endfor %}
+{%     set indent = '    ' %}
```

Comment:
Why introduce this if it's not used anywhere else in the template? Makes a simple thing overly complicated, IMHO :)

Reply:
It's a general habit of mine to put everything in a variable when it's used more than once. Might be overkill here, although it would be easy to change when used throughout the entire template.

Reply:
I've removed the `indent` variable. I did put `join('\n' + '    ')` to make the indentation more explicit, as this should not be removed.